dotnet/src/Connectors/Connectors.AzureAIInference/Core/ChatClientCore.cs

```diff
@@ -110,7 +110,8 @@ internal ChatClientCore(
         this.Logger = logger ?? NullLogger.Instance;
         // Accepts the endpoint if provided, otherwise uses the default Azure AI Inference endpoint.
         this.Endpoint = endpoint ?? httpClient?.BaseAddress;
-        Verify.NotNull(this.Endpoint);
+        Verify.NotNull(this.Endpoint, "endpoint or base-address");
```

Comment:
Should this be `httpClient.BaseAddress`?

Reply:
No, otherwise the error would be misleading, where the caller can solve it by providing the `endpoint` also not just the `httpClient.BaseAddress`.